roles/common/tasks/awscli.yml

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
What do you think about this approach?

```suggestion
```yaml
- name:
  pip:
    name: awscli
```
```

Reply:
awscli v2 my dear